### DISCUSSION TITLE
[FABRIC] Anyone else having trouble installing the 1.20.1 version ?

### DISCUSSION THREAD

**Discussion by ItsaiKara** [bernie-g/geckolib#468] [Q&A] [answered]
I tried to download it from https://dl.cloudsmith.io/public/geckolib3/geckolib/maven/ in my build.gradle but I can't seem to download the lib is the link dead ?
Discussed on discord here: 
https://discord.com/channels/507304429255393322/507982478276034570/1134897043236737205

**Comment by ItsaiKara** [comment, marked as answer]
NVM, the link I thought was outdated was good. I'm using Vscode and in order to get the dependencies I had to restart my whole computer since gradle ui is kinda bad

**Reply by dombomb64** [reply]
What did you do besides restarting? I restarted, and the problem is still there for me.
